src/app/components/ebi-submission-menu/ebi-submission-menu.component.html

```diff
@@ -0,0 +1,5 @@
+<ul class="tabs">
+  <li *ngFor="let link of tabLinks" routerLinkActive="active is-active" class="tabs-title">
+    <a [class.disabled]="isTabsDisabled()" [routerLink]="link.href" routerLinkActive="active">{{link.title}}</a>
```

Comment:
Would it be possible to change `isTabsDisabled()` for a property?

Reply:
This is fixed